packages/cli/README.md

```
@@ -0,0 +1,96 @@
+
+## Project Status
+
+This repository is not actively maintained. It's recommended to use one of the third-party clients that support the JSON Resume standard instead:
+
+* [Resumed](https://github.com/rbardini/resumed)
+
+## Getting Started
+
+Install the command-line tool:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)